### ISSUE TITLE
Distributed cache is not updated

### ISSUE THREAD

**Issue by RobertCopilau** [umbraco/Umbraco-CMS#4513]
Happens after installing The-Starter-Kit package.

```2019-02-11 08:59:35,917 [P5820/D8/T8] ERROR  Umbraco.Core.Sync.DatabaseServerMessenger - DISTRIBUTED CACHE IS NOT UPDATED. Failed to execute instructions (2: '[{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1048,\"Key\":\"135d60e0-64d9-49ed-ab08-893c9ba44ae5\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1060,\"Key\":\"3ece86aa-61ad-45d5-b1dd-8f02f25df949\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1061,\"Key\":\"71548df5-836c-41f8-96ac-5b98fe5e2e19\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1062,\"Key\":\"34a7f8f5-a84e-439b-9322-466ff7ed8866\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1063,\"Key\":\"fff3c360-3f90-45b3-a554-e29ca72cdce4\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1064,\"Key\":\"519eaa41-b905-4371-aa7c-40c9b3946f66\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1065,\"Key\":\"9d5ba2c5-ed7a-41f8-b454-9fc65f48752e\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1066,\"Key\":\"9d0726e6-5fa7-4455-8fe6-ed5bf9057cc4\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1067,\"Key\":\"ecbbac0d-974e-490c-b533-fead926dc525\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1068,\"Key\":\"ea0ed4c1-c489-43b8-9058-a70babf430ff\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1069,\"Key\":\"4885450e-a60f-42bb-984a-43988baf5283\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1070,\"Key\":\"b58c1530-91c9-4c83-aa35-032aca0f7b89\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1071,\"Key\":\"e26a8d91-a9d7-475b-bc3b-2a09f4743754\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1072,\"Key\":\"59ac0a33-9d34-4236-9886-99310d13d7f1\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1073,\"Key\":\"30c35221-91b0-49a5-a599-42d1965161ea\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1074,\"Key\":\"84bbeb67-df7e-4043-8db8-55df52d01456\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1075,\"Key\":\"3ada047c-6dfc-4a14-afc4-7efb79390f92\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1076,\"Key\":\"3f6619b6-08a4-4be7-8d6a-2761844ee567\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1077,\"Key\":\"28a6aeda-b3fc-4c4d-926e-607854a07891\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"35b16c25-a17e-45d7-bc8f-edab1dcc28d2","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1078,\"Key\":\"d0382188-119b-49b7-86d3-84119a02645f\",\"Removed\":false}]"},{"RefreshType":4,"RefresherId":"7b1e683c-5f34-43dd-803d-9699ea1e98ca","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1,\"Alias\":\"latestBlogposts\"}]"},{"RefreshType":4,"RefresherId":"7b1e683c-5f34-43dd-803d-9699ea1e98ca","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":2,\"Alias\":\"featuredProduct\"}]"},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1079]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1080]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1081]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1082]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1083]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1084]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1085]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1086]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1087]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1088]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":4,"RefresherId":"6902e22c-9c10-483c-91f3-66b7cae9e2f5","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"ItemType\":\"IContentType\",\"Id\":1090,\"ChangeTypes\":1},{\"ItemType\":\"IContentType\",\"Id\":1091,\"ChangeTypes\":1},{\"ItemType\":\"IContentType\",\"Id\":1092,\"ChangeTypes\":1},{\"ItemType\":\"IContentType\",\"Id\":1093,\"ChangeTypes\":1},{\"ItemType\":\"IContentType\",\"Id\":1094,\"ChangeTypes\":1},{\"ItemType\":\"IContentType\",\"Id\":1095,\"ChangeTypes\":1},{\"ItemType\":\"IContentType\",\"Id\":1096,\"ChangeTypes\":1},{\"ItemType\":\"IContentType\",\"Id\":1097,\"ChangeTypes\":1},{\"ItemType\":\"IContentType\",\"Id\":1098,\"ChangeTypes\":1},{\"ItemType\":\"IContentType\",\"Id\":1099,\"ChangeTypes\":1},{\"ItemType\":\"IContentType\",\"Id\":1100,\"ChangeTypes\":1},{\"ItemType\":\"IContentType\",\"Id\":1101,\"ChangeTypes\":1}]"},{"RefreshType":4,"RefresherId":"6902e22c-9c10-483c-91f3-66b7cae9e2f5","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"ItemType\":\"IContentType\",\"Id\":1092,\"ChangeTypes\":4},{\"ItemType\":\"IContentType\",\"Id\":1095,\"ChangeTypes\":4},{\"ItemType\":\"IContentType\",\"Id\":1096,\"ChangeTypes\":4},{\"ItemType\":\"IContentType\",\"Id\":1098,\"ChangeTypes\":4},{\"ItemType\":\"IContentType\",\"Id\":1101,\"ChangeTypes\":4}]"},{"RefreshType":4,"RefresherId":"900a4fbe-df3c-41e6-bb77-be896cd158ea","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1102,\"ChangeTypes\":2},{\"Id\":1103,\"ChangeTypes\":2},{\"Id\":1104,\"ChangeTypes\":2},{\"Id\":1105,\"ChangeTypes\":2},{\"Id\":1106,\"ChangeTypes\":2},{\"Id\":1107,\"ChangeTypes\":2},{\"Id\":1108,\"ChangeTypes\":2},{\"Id\":1109,\"ChangeTypes\":2},{\"Id\":1110,\"ChangeTypes\":2},{\"Id\":1111,\"ChangeTypes\":2},{\"Id\":1112,\"ChangeTypes\":2},{\"Id\":1113,\"ChangeTypes\":2},{\"Id\":1114,\"ChangeTypes\":2},{\"Id\":1115,\"ChangeTypes\":2},{\"Id\":1116,\"ChangeTypes\":2},{\"Id\":1117,\"ChangeTypes\":2},{\"Id\":1118,\"ChangeTypes\":2},{\"Id\":1119,\"ChangeTypes\":2},{\"Id\":1120,\"ChangeTypes\":2},{\"Id\":1121,\"ChangeTypes\":2},{\"Id\":1122,\"ChangeTypes\":2},{\"Id\":1123,\"ChangeTypes\":2},{\"Id\":1124,\"ChangeTypes\":2},{\"Id\":1125,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"6902e22c-9c10-483c-91f3-66b7cae9e2f5","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"ItemType\":\"IContentType\",\"Id\":1099,\"ChangeTypes\":4}]"},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1081]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1079]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1080]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1081]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1082]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1083]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1085]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1086]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1087]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":3,"RefresherId":"dd12b6a0-14b9-46e8-8800-c154f74047c8","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":"[1088]","JsonIdCount":1,"JsonPayload":null},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1126,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1127,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1128,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1129,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1130,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1131,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1132,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1133,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1134,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1135,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1136,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1137,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1138,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1139,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1140,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1141,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"b29286dd-2d40-4ddb-b325-681226589fec","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1142,\"ChangeTypes\":2}]"},{"RefreshType":4,"RefresherId":"900a4fbe-df3c-41e6-bb77-be896cd158ea","GuidId":"00000000-0000-0000-0000-000000000000","IntId":0,"JsonIds":null,"JsonIdCount":1,"JsonPayload":"[{\"Id\":1102,\"ChangeTypes\":4}]"}]'). Instruction is being skipped/ignored
System.NullReferenceException: Object reference not set to an instance of an object.
   at Umbraco.Web.Routing.RedirectTrackingComponent.get_OldRoutes()
   at Umbraco.Web.Routing.RedirectTrackingComponent.ContentCacheRefresher_CacheUpdated(ContentCacheRefresher sender, CacheRefresherEventArgs args)
   at Umbraco.Core.Events.TypedEventHandler`2.Invoke(TSender sender, TEventArgs e)
   at Umbraco.Web.Cache.ContentCacheRefresher.Refresh(JsonPayload[] payloads)
   at Umbraco.Core.Sync.DatabaseServerMessenger.NotifyRefreshers(IEnumerable`1 instructions, HashSet`1 processed)
   at Umbraco.Core.Sync.DatabaseServerMessenger.ProcessDatabaseInstructions(IReadOnlyCollection`1 instructionBatch, CacheInstructionDto dto, HashSet`1 processed, Int32& lastId)
2019-02-11 08:59:35,923 [P5820/D8/T8] INFO   Umbraco.Core.Sync.DatabaseServerMessenger - The current batch of instructions was not processed, app is shutting down
2019-02-11 08:59:36,002 [P5820/D8/T6] WARN   Umbraco.Core.Scoping.ScopeProvider - Missed ScopeContext Object 929ac5ea``

**Comment by RobertCopilau** [comment]
[The-Starter_Kit_4.0.0.zip](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5b151a952d7cd20011c4533d/125516c66e82c728ace21e0d46dc9c9f6eb197a89dfdba519b76f980161269236754e1673295f127b54778154d555ca9130d5d10fbfa8c7ff0b23f758a5c0aecd4680a7d926b14bea4905dee5fee7fd616cfe3fe9846d2e93a2fe68e8b46361907108bfc5d33a08b06.zip)

**Comment by zpqrtbnk** [comment]
Cannot repro, but obviously the redirect tracking was not wired correctly, should be fixed with 154b59c84b

**Comment by RobertCopilau** [comment]
Yep, seems to be fixed - I have not encountered the issue anymore.
Closing this down.
